src/main/resources/emails/rollup.ftlh

```diff
@@ -184,7 +184,7 @@ a {
         </div>
     </div>
 
-    <a href="${serverUrl}/reports/${report.uuid}/min" >
+    <a href="${serverUrl}/reports/${report.uuid}" >
```

Comment:
Maybe we could link Print Report page (pages/reports/Compact.js at https://github.com/NCI-Agency/anet/pull/3161 ) here in future. Or maybe even better, turn it into pdf and link that pdf.